Gemfile

```diff
@@ -25,5 +25,6 @@ group :jekyll_plugins do
 end
 
 gem "webrick", "~> 1.7"
-gem 'csv'
-gem 'logger'
+gem "csv"
+gem "logger"
+gem "faraday-retry"
```

Comment:
What is `faraday-retry` and why is it being added here?

Reply:
`faraday-retry` is a good library for handling errors and exceptions and will retry up to 2 times if the request fails due to server issues or other issues. I remember when developing and testing this PR, I encountered some problems and ruby ​​recommended that I add this library, so I did.
If you want, I can remove it now.

Best